docs_src/design/ucr/0004-Provision-Watch-more-Device-Properties.md

```diff
@@ -0,0 +1,78 @@
+## Provision Watch more Device Properties
+This UCR describes the Use Case for Provision Watching via Additional Device Properties,
+beyond the protocol properties currently used exclusively for matching in Provision Watchers.
+
+### Submitters
+- Tom Brennan (Eaton)
+
+## Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pulls) (2022-07-26)
+
+
+### Market Segments
+Any that deploy EdgeX systems with analytic, utility, or north-bound microservices that must "discover"
+Devices added to the EdgeX core-metadata by south-bound Device Services.
+
+### Motivation
+The autodiscovery of Devices using Provision Watchers is a useful feature of Device Services; currently,
+the Provision Watcher implementation in the two Device SDKs uses the protocol properties of a discovered
+Device to match against the "identifiers" specified in the Provision Watcher metadata. The 
+implementations use regular expression matching against the "identifiers", and also filter out any 
+Devices whose protocol properties match the "blockingIdentifiers" of the Provision Watcher metadata.
+
+We are finding that [Hybrid App-Device Services](https://github.com/edgexfoundry/edgex-docs/pulls) *later (./0005-Hybrid-App-Device-Services.md)* also want to 
+use Provision Watchers, so that they can be configured at run-time to work with new Devices, but these 
+do not need or want to match the protocol properties of a Device; instead, they want to match or exclude
+based on Device properties such as the "profileName" (as a proxy for Device Model Name), "name", and "labels".
+
+This UCR describes the Use Case for using these additional properties for Provision Watching.
+
+### Target Users
+- Product Developers
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud User
+- Service Provider
+- Software Integrator
+
+### Description
+Application Services using the Device SDKs (ie, Hybrid App-Device Services) can take advantage of the
+Provision Watching feature and APIs to "discover" new EdgeX devices from the south-bound Device 
```

Comment:
Confused on why the Hybrid App service would need to re-discover devices that the south-bound Device Services have already discovered, statically added or where manually added. Seems to me the App service should use the new Device System Events to know when a new Device has been added. It can use subscription to filter for some of what you want to add to the Provision Watcher. See ADR here: https://docs.edgexfoundry.org/2.3/design/adr/0024-system-events/#messagebus-topic

Reply:
DSE isn't just for dynamic part of Device Discovery. It is when any device is added/update/delete weather from static files or from Device Discovery. The service receiving DSEs must be up and ready before Core Metadata which publishes the DSEs during initialization.

Reply:
@lenny-intel OK to mark this comment Resolved now?